Config/BuildSettings.swift

```diff
@@ -426,8 +426,7 @@ final class BuildSettings: NSObject {
         guard self.locationSharingEnabled else {
             return false
         }
-        
-        // Do not enable live location sharing atm
-        return false
+
```

Comment:
We are just returning ```locationSharingEnabled``` here then? I think we can simplify greatly the entire LLS section here, as iOS 14 is now the minimal target. 

I also think naming is a bit misleading here, if I understand correctly what we want to achieve, BuildSettings should contain 2 distinct bool values:
* whether the feature is available at all in this build
* the default value of the UserDefault

Reply:
Good catch both! I removed the iOS 14 checks and unified the two build settings into one. I chose to not create a dedicated setting for the `UserDefaults` default value because we always want to keep it disabled by default for now due to the fact that location history persists in the timeline. @MaximeEvrard42 is working on a promotional treatment that will make it possible to flip the labs flag directly from the composer.